truss/templates/server/model_wrapper.py

```diff
@@ -253,6 +274,12 @@ async def __call__(
             Generator: In case of streaming response
         """
 
+        if self.truss_schema is not None:
+            try:
+                body = self.truss_schema.input_type(**body)
+            except pydantic.ValidationError as e:
+                raise HTTPException(status_code=400, detail=str(e))
```

Comment:
What happens with this `HTTPException`? I wanna make sure that we're not leaking exception details over the wire. The exception `e` should be logged so the developer can debug, but shouldn't be returned as part of the response.

Reply:
It will return the validation details (ie: what fields you're missing from the request). We can return something more generic here and log the exception, that's fine.